stl/inc/atomic

```diff
@@ -1109,7 +1109,7 @@ struct _Atomic_storage<_Ty, 8> { // lock-free using 8-byte intrinsics
     }
 
     _NODISCARD _TVal load(const memory_order _Order) const noexcept { // load with given memory order
-        const auto _Mem = _Atomic_address_as<long long>(_Storage);
+        const auto _Mem = _STD _Atomic_address_as<long long>(_Storage);

```

Comment:
There are 32-bit test failures, involving line 1001 above:

```
D:\a\_work\1\s\tests\std\tests\GH_001596_adl_proof_algorithms\test.compile.pass.cpp(30,7): error: field has incomplete type 'incomplete'
    T t;
      ^
D:\build\out\inc\atomic(1001,27): note: in instantiation of template class 'holder<incomplete>' requested here
        const auto _Mem = _Atomic_address_as<int>(_Storage);
                          ^
D:\build\out\inc\execution(373,24): note: in instantiation of member function 'std::_Atomic_storage<holder<incomplete> **, 4>::load' requested here
        return _Result.load(memory_order_relaxed) != _Last;
                       ^
[...]
```

I'm assuming that the 4-byte code path has to be altered for 32-bit pointers, similarly to the 8-byte code path for 64-bit pointers here.